packages/bits/src/lib/repeat/repeat.component.ts

```diff
@@ -313,7 +360,7 @@ export class RepeatComponent<T extends IRepeatItem = any> implements OnInit, OnD
             type: nameof<INovaFilters>("selection"),
             value: {
                 selection: this.selection,
-                itemsSource: this.itemsSource,
+                itemsSource: this.itemsSource.slice(),
```

Comment:
is this change needed?

Reply:
actually leftover from change I then removed as "leave for v12" (changing of input signature to readonly array so we do not require modifiable array for itemsSource, b/c we have no reason to manipulate the array given to us, ideally all the types should change but that would be getting too ahead). I can remove this as well and leave for the v12 change